k8s/install/terraform/terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+ssh_key_file = "<ssh_public_key_file_path>"
+ssh_private_key_file = "<ssh_public_key_file_path>"
+# zCompute API IP/Host address e.g.:
+# zcompute_api_ip = "zcompute.cloud.zadara.net"
+zcompute_api_ip = "<zcompute host name>"
+# A Centos 7.9 AMI id available to the project
```

Comment:
maybe you can add in the readme me that  the image is available through the marketplace